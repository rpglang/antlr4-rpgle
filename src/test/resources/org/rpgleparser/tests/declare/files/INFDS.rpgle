     FCOLMBID   UF   E           K DISK    INFDS(DSMBID) INFSR(SUBRER) 
     F                                     RENAME(COPMB1:COLMBID1)  