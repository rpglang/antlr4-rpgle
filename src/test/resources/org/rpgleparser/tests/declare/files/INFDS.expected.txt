/*===TOKENS===*/
FS_FIXED                 <F>
FS_RecordName            <COLMBID   >
FS_Type                  <U>
FS_Designation           <F>
FS_EndOfFile             < >
FS_Addution              < >
FS_Sequence              < >
FS_Format                <E>
FS_RecordLength          <     >
FS_Limits                < >
FS_LengthOfKey           <     >
FS_RecordAddressType     <K>
FS_Organization          < >
FS_Device                <DISK   >
FS_Reserved              < >
KEYWORD_INFDS            <INFDS>
'('                      <(>
ID                       <DSMBID>
')'                      <)>
KEYWORD_INFSR            <INFSR>
'('                      <(>
ID                       <SUBRER>
')'                      <)>
KEYWORD_RENAME           <RENAME>
'('                      <(>
ID                       <COPMB1>
COLON                    <:>
ID                       <COLMBID1>
')'                      <)>
/*===TREE===*/
(r
  (statement
    (fspec_fixed
      F
      'COLMBID   '
      U
      F
      ' '
      ' '
      ' '
      E
      '     '
      ' '
      '     '
      K
      ' '
      'DISK   '
      ' '
      (fs_keyword
        (keyword_infds
          INFDS
          (
          (simpleExpression (identifier (free_identifier (idOrKeyword DSMBID))))
          )
        )
      )
      (fs_keyword
        (keyword_infsr
          INFSR
          (
          (simpleExpression (identifier (free_identifier (idOrKeyword SUBRER))))
          )
        )
      )
      (fs_keyword
        (keyword_rename
          RENAME
          (
          (simpleExpression (identifier (free_identifier (idOrKeyword COPMB1))))
          :
          (simpleExpression (identifier (free_identifier (idOrKeyword COLMBID1))))
          )
        )
      )
      <EOF>
    )
  )
)
/*======*/